.github/workflows/static-analysis.yml

```diff
@@ -34,6 +34,35 @@ jobs:
         gradle-home-cache-cleanup: true
     - name: Check copyrights, license headers, and .gitattributes
       run: ./gradlew checkCopyrightsInNoticeFile checkLicenseHeaders checkGitAttributes
+  completions:
+    runs-on: ubuntu-22.04
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4
+    - name: Setup Gradle
+      uses: gradle/actions/setup-gradle@af1da67850ed9a4cedd57bfd976089dd991e2582 # v4
+      with:
+        gradle-home-cache-cleanup: true
+    - name: Generate completions
+      run: |
+        ./gradlew -q :cli:run --args="--generate-completion=bash" > integrations/completions/ort-completion.bash
+        ./gradlew -q :cli:run --args="--generate-completion=fish" > integrations/completions/ort-completion.fish
+        ./gradlew -q :cli:run --args="--generate-completion=zsh" > integrations/completions/ort-completion.zsh
+        # Renormalize the completions to fix the line endings.
+        git add --renormalize integrations/completions
+        git reset
```

Comment:
I'm surprised this is needed. Gradle runs on Linux, and also the clikt option should write out Linux line endings.

Anyway, could we instead use `git diff --exit-code` in line 56 instead? Eventually with `--ignore-cr-at-eol`?

Reply:
Coincidentally, I just did that change. I'll turn that back into a draft, because if the check succeeds now, I'll also have to verify that it fails when it should.

Reply:
@sschuberth Still doesn't work, any idea what's wrong? When I test the if condition locally it works.

Reply:
The [diff](https://github.com/oss-review-toolkit/ort/actions/runs/10549414492/job/29224431884#step:5:23) shows interesting part of Gradle build output, but I'm not sure yet why.

Reply:
I guess the `-q` from Gradle doesn't apply to the Vite build. That would explain why it works locally, because I had built that module before. I'll try to separate building from running, that should solve it if the assumption is correct.

Reply:
Now it works.